src/plugins/dashboard/public/attribute_service/attribute_service.tsx

```diff
@@ -204,7 +204,9 @@ export class AttributeService<
           const newAttributes = { ...input[ATTRIBUTE_SERVICE_KEY] };
           newAttributes.title = props.newTitle;
           const wrappedInput = (await this.wrapAttributes(newAttributes, true)) as RefType;
-          resolve(wrappedInput);
+
+          // Combine input and wrapped input to preserve any passed in explicit Input.
+          resolve({ ...input, ...wrappedInput });
```

Comment:
This is to preserve explicit input like panel title or time range. Otherwise it would get lost in translation when going between 'by reference' and 'by value' and back.